docker-builds/reference-crucible/Dockerfile

```diff
@@ -19,6 +19,10 @@ RUN apt-get install -y mongodb mongodb-server && \
 RUN mkdir -p /etc/service/mongodb/
 ADD mongodb.sh /etc/service/mongodb/run
 
+# Set up delayed jobs
+ADD delayed-job /etc/init.d/delayed-job
+RUN update-rc.d delayed-job defaults
```

Comment:
Does this work? When is the job invoked at runtime? I don't see this approach described at https://github.com/phusion/baseimage-docker